features/bevragen/overzicht.md

```diff
@@ -0,0 +1,80 @@
+# Personen bevragen
+
+De BRP bevragen API maakt het mogelijk om gegevens van actuele personen in de basisregistratie personen (BRP) te raadplegen. De personen worden opgezocht op basis van hun identificerende gegevens.
+
+In de BRP wordt een persoon uniek geïdentificeerd met behulp van hun burgerservicenummer. Is de burgerservicenummer van de te raadplegen persoon/personen bekend, dan moet de [Raadpleeg persoon met burgerservicenummer](#raadplegen-van-personen) operatie worden gebruikt om de betreffende persoon/personen te raadplegen.
```

Comment:
`In de BRP wordt een persoon uniek geïdentificeerd ...` --> `In de BRP worden personen uniek geïdentificeerd ...`

`Is de burgerservicenummer ...` --> `Is het burgerservicenummer ...`

Reply:
aangepast